### DISCUSSION TITLE
whats the difference between the auth and gotrue docker images?

### DISCUSSION THREAD

**Discussion by crackedbeefcake** [supabase/supabase#21364] [Questions] [answered]
hi
https://hub.docker.com/r/supabase/auth
https://hub.docker.com/r/supabase/gotrue

there are two docker images that are solving the same problem it seems. whats the difference?

**Comment by GaryAustin1** [comment, marked as answer]
From last launch week blog:


`If you’ve been following us from [the start](https://supabase.com/blog/supabase-auth), you will know that Supabase Auth started by forking [Netlify’s GoTrue server](https://github.com/netlify/gotrue). A lot has changed since then and we’ve diverged from the upstream repository. At this stage it makes sense to rename the project to something else (cues drumroll) — Auth.`

`This simply means that the repositories will be renamed from using gotrue to auth. But don’t worry! Docker images and libraries like @supabase/gotrue-js will continue to be published and you can use @supabase/auth-js interchangeably for the current v2 version for as long as it is supported. All of the classes and methods remain in place. No breaking changes here!`
